roles/as_code/lib/technical_principal.rb

```diff
@@ -0,0 +1,79 @@
+
+    Other notable things:
+
+    - Every Friday afternoon is dedicated to learning new skills
+    - Everyone is encouraged to write blog posts regularly
+    - Our handbook is open sourced
+    - We are vegan and non-drinker friendly as well as meat-eater and drinker friendly
+    - Retreats and trips every year
+
+    ## Salary
+
+    This role has a salary of £85-95k depending on experience.
+  DESCRIPTION
+
+  salary 85_000..95_000
+
+  expected 'to act as a role model and servant leader to their teams',
+    'We expect our Technical Principals to lead by example, to put the needs of others before their own, and to set and demonstrate a high standard for code and delivery quality. A Technical Principal is expected to show themselves to be highly reliable, to be on time, and to be well prepared.'
+
+  expected 'to coach and nurture Lead and Senior Engineers',
+    'We expect our Technical Principals to proactively provide thoughtful and meaningful feedback for their team. A Technical Principal is expected to spend time helping team members to improve their skills. A Technical Principal is expected to identify and nurture candidates for Technical Principal positions. A Technical Principal is expected to identify and escalate performance issues to a relevant Line Manager.'
+
+  expected 'to be a point of escalation for software engineering issues',
+    'We expect our Technical Principals to be a point of escalation when a delivery team are struggling with software engineering issues. Technical Principals will look to quickly remedy issues and pull in the wider engineering team if necessary.'
+
+  expected 'to coach and support teams running workshops and roadmapping sessions',
+    'We expect our Technical Principals to coach teams to be able to run workshops and roadmapping sessions, to provide a supporting role in important sessions, and foster collaboration with the wider team to identify a technology roadmap to solve business problems.'
+
+  expected 'to run technical discovery projects',
+    'We expect our Technical Principals to run technical discoveries. Technical Principals are responsible for planning, workshopping, running technical feasibility studies, and ultimately recommending strategies to customers.'
+
+  expected 'to be partner with customer tech leadership (CTO/CIO) to help shape their technology strategies',
+    'We expect our Technical Pricipals to be able to engage the highest levels of technical leadership within our customers organisations. Through building strong relationships Technical Principals will be seen as trusted advisors.'
+
```

Comment:
These expectations feel like they'd be the bulk of the time spent - on interfacing with the customer.